echologue-frontend/src/index.css

```
@@ -2,68 +2,274 @@
   .btn {
-    @apply px-4 py-2 rounded-lg font-medium transition-all duration-200;
-    @apply hover:shadow-lg active:scale-95;
+    @apply inline-flex items-center justify-center px-4 py-2 rounded-md font-medium text-sm
+           transition-all duration-200 cursor-pointer select-none;
   }
-  
+
   .btn-primary {
-    @apply bg-primary text-white hover:bg-primary-dark;
+    @apply bg-ink-accent hover:opacity-90 active:scale-95;
+    color: var(--color-base);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)